index.html

```diff
@@ -0,0 +1,110 @@
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section class="header">
+      <a class="name" href="assets/images/Vahan.jpg"
+        ><p>Vahan Khachvankian</p></a
+      >
+      <nav class="burger">
+        <a href="#"><img src="assets/images/lines-56763.png" alt="logo" /></a>
+      </nav>
+    </section>
+
+    <section class="main">
+      <div class="title">
+        <h1>
+          Hey There. <br />
+          I'm Vahan
+        </h1>
+      </div>
+      <div class="head">
+        <p class="pretext">I'am a Software Developer</p>
+      </div>
+      <div class="content">
+        <p class="text">
+          I can help you build a product, feature or website. Look through of
+          some my work and experience! If you like what you see and have a
+          project you need coded, don't hesitate to contact me.
+        </p>
+      </div>
+
+      <nav class="navigator">
+        <ul id="menu">
+          <li>
+            <a
+              href="https://github.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="github height"
+                src="assets/images/github.svg"
+                alt="github logo"
+            /></a>
+          </li>
+          <li class="bg height">
+            <a
+              href="https://www.vectorstock.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="vector-one height"
+                src="assets/images/Vector.png"
+                alt="vector-one" />
+              <img
+                class="vector-two height"
+                src="assets/images/VectorN.png"
+                alt="vector-two"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://angel.co/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="angel height"
+                src="assets/images/angellist.svg"
+                alt="angel logo"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://twitter.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="twitter height"
+                src="assets/images/twitter.svg"
+                alt="twitter logo"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://medium.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="medium height"
+                src="assets/images/medium.svg"
+                alt="medium logo"
+            /></a>
+          </li>
+        </ul>
+        <div class="background">
+          <img class="bottom" src="assets/images/VectorZet.png" alt="bottom" />
+        </div>
```

Comment:
- [ ] As stated by the previous reviewer, kindly make sure all page decorations are background image, not an image with `img` tag as stated in the project requirement😉

Reply:
Thank you. 
But again, I have some doubts. You write that the previous reviewer asked for this change, but she does not have such a requirement. Therefore, the form in which your requirement is presented here creates the impression that I did not fulfill the requirement of the previous reviewer, which does not correspond to reality.

Reply:
Hi again😊,
Kindly see the below screenshot for the review for the previous reviewer✌, I think you might have missed it I'm sorry if it causes any mix-up or confusion🙏
 
![screenshot (40)](https://user-images.githubusercontent.com/44624138/144005065-a1b8649b-8d80-42c2-9cb1-8d1b8f5240eb.png)

Reply:
O.K. Something wrong with my page. I don't see this picture from Juliana. Thank you, for you sent it to me.

Reply:
Replaced.